### PR TITLE
Chapter 2

### PR DESCRIPTION
**Pull request by asad5059**
Chapter 2 is done for now.

### CODE REVIEW

File: Chapter 2/README.md
Comment (towfiq-ul) [inline Chapter 2/README.md:1]: this `README.md` file should be in `Intro-to-Java-Beginner-to-Advance/README.md` path. 
This `Chapter 2/README.md` should contain different data in it.

**Review by towfiq-ul — CHANGES_REQUESTED** [review]
Please read the comment(s) and adopt necessary changes.

File: Chapter 2/README.md
Comment (asad5059) [reply Chapter 2/README.md:1]: Necessary changes are made. Please review again bhaiya.

File: README.md [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-The repo is organized by chapter and section, with each section containing my summaries of the material covered in the book, along with examples and exercises to practice what I've learned. I can simply navigate to the section I want to study and start exploring.
-
-## Contributing
-
-Since this repository is for my personal use, contributions are not necessary. However, if you have any suggestions or feedback, feel free to contact me.
-
-## Acknowledgments
-
-Special thanks to Herbert Schildt for writing "Java: The Complete Reference" and to the open-source community for their contributions to the Java programming language.
-
```
Comment (towfiq-ul) [inline README.md]: You removed this file from the project. So after merging this file will be remove from the repository and you will not find it in the parent dir.  So, resolve it.

**Review by towfiq-ul — CHANGES_REQUESTED** [review]
resolve the requested changes.

**Review by towfiq-ul — APPROVED** [review]
LGTM

**Pull request merged**